src/platform/plugins/shared/discover/public/application/main/components/loading_spinner/loading_spinner.tsx

```diff
@@ -8,26 +8,27 @@
  */
 
 import React from 'react';
-import type { UseEuiTheme } from '@elastic/eui';
-import { EuiLoadingSpinner, EuiTitle, EuiSpacer } from '@elastic/eui';
+import { EuiLoadingSpinner, EuiTitle, EuiSpacer, useEuiTheme } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { css } from '@emotion/react';
 
 export function LoadingSpinner() {
-  const loadingSpinnerCss = ({ euiTheme }: UseEuiTheme) => css`
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    text-align: center;
-    padding: ${euiTheme.size.l} 0;
-    background-color: ${euiTheme.colors.backgroundBasePlain};
-    z-index: 3;
-  `;
-
```

Comment:
Unit tests print console.error for this usage, however those style work in Kibana and tests pass anyways. I moved it to clear console errors.